### PR TITLE
Added Russian translation

### PR DESCRIPTION
**Pull request by SyberiaKey**
One thing that must be pointed out: i didn't find where "text.autoconfig.dynamiccrosshair.option.dynamicCrosshairEnableWithHud" string is used so i tried my best to translate it fine w/o context.

### CODE REVIEW

**Comment by SyberiaKey** [comment]
Also want to mention that i'm not quite sure where i can find these strings
изображение

**Comment by Crendgrim** [comment]
Oh neat, thank you!

Those "strings" on the buttons are the internal value names that get automatically used. I honestly couldn't figure out how to change them manually to make them translatable. I'll have to look into that again now...

File: src/main/resources/assets/dynamiccrosshair/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingTool": "Держит в руках инструмент",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingRangedWeapon": "Держит в руках оружие дальнего боя",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingThrowable": "Держит кидаемый предмет",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingBlock": "Держит в руках блок",
+  "text.autoconfig.dynamiccrosshair.option.crosshairStyle.holdingUsableItem": "Держит в руках используемый предмет",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers": "Модификаторы прицела",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modInteractable": "Смотрит на интерактивный блок",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modCorrectTool": "Держит в руках подходящий инструмент",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modIncorrectTool": "Держит в руках неподходящий инструмент",
+  "text.autoconfig.dynamiccrosshair.option.crosshairModifiers.modUsableItem": "Holding usable item"
```
Comment (Crendgrim) [inline src/main/resources/assets/dynamiccrosshair/lang/ru_ru.json:30]: Would you mind adding a translation to this one key that is still in English before I merge? :)

**Comment by SyberiaKey** [comment]
Ok, one minute

вс, 22 мая 2022 г., 02:44 Crendgrim ***@***.***>:

**Comment by Crendgrim** [comment]
Thank you!

**Comment by Crendgrim** [comment]
It turns out that it was much easier than expected. Current `main` supports translating those buttons if you'd like to finish the translation :)

**Pull request merged**